packages/pn-personagiuridica-webapp/public/locales/it/notifiche.json

```diff
@@ -67,7 +67,10 @@
     "add_domicile_EMAIL": "Aggiungi e-mail",
     "domicile_PEC": "Hai un indirizzo PEC? Aggiungilo e non riceverai più le raccomandate cartacee.",
     "domicile_EMAIL": "Aggiungi il tuo indirizzo e-mail: ti invieremo un messaggio ogni volta che ricevi una notifica.",
+    "cancelled-alert-text": "Questa notifica è stata annullata dall’ente mittente. Puoi ignorarne il contenuto.",
```

Comment:
Why there aren't the localizations about the download of the documents?

Check also the old localizations and remove them

Reply:
do it